src/other/Table.js

```diff
@@ -17,36 +17,69 @@
+                [41.244123, -95.961610, "", 4525, 423545, 247354, 54, 2552345, 2455, 245435],
+                [32.688980, -117.192040, "", 4525, 423545, 243654, 54, 2552345, 2455, 245435],
+                [45.786490, -108.526600, "", 4525, 423545, 245354, 54, 2552345, 2455, 245435],
+                [45.796180, -108.535652, "", 4525, 423545, 243354, 54, 2552345, 2455, 245435],
+                [45.774320, -108.494370, "", 4525, 423545, 243454, 54, 2552345, 2455, 245435],
+                [37.665074, -122.384375, "green-dot.png", 4525, 423545, 243254, 54, 2552345, 2455, 245435],
+                [32.690680, -117.178540, "", 4525, 4243545, 24354, 54, 2552345, 2455, 245435],
+                [39.709455, -104.969859, "", 4525, 4233545, 24354, 54, 2552345, 2455, 245435],
+                [41.244123, -95.961610, "", 4525, 4235145, 24354, 54, 2552345, 2455, 245435],
+                [32.688980, -117.192040, "", 4525, 4523545, 24354, 54, 2552345, 2455, 245435],
+                [45.786490, -108.526600, "", 4525, 4263545, 24354, 54, 2552345, 2455, 245435],
+                [45.796180, -108.535652, "", 4525, 4235745, 24354, 54, 2552345, 2455, 245435],
+                [45.774320, -108.494370, "", 4525, 4235845, 24354, 54, 2552345, 2455, 245435],
+                [37.665074, -122.384375, "green-dot.png", 4525, 423545, 24354, 54, 2552345, 2455, 245435],
+                [32.690680, -117.178540, "", 4525, 423545, 24354, 54, 2552345, 20455, 245435],
+                [39.709455, -104.969859, "", 4525, 423545, 24354, 54, 2552345, 24955, 245435],
+                [41.244123, -95.961610, "", 4525, 423545, 24354, 54, 2552345, 24855, 245435],
+                [32.688980, -117.192040, "", 4525, 423545, 24354, 54, 2552345, 27455, 245435],
+                [45.786490, -108.526600, "", 4525, 423545, 24354, 54, 2552345, 24655, 245435],
+                [45.796180, -108.535652, "", 4525, 423545, 24354, 54, 2552345, 24555, 245435],
+                [45.774320, -108.494370, "", 4525, 423545, 24354, 54, 2552345, 24455, 245435],
+                [37.665074, -122.384375, "green-dot.png", 4525, 423545, 24354, 54, 23552345, 2455, 245435],
+                [32.690680, -117.178540, "", 4525, 423545, 24354, 54, 2552345, 2455, 2405435],
+                [39.709455, -104.969859, "", 4525, 423545, 24354, 54, 2552345, 2455, 2495435],
+                [41.244123, -95.961610, "", 4525, 423545, 24354, 54, 2552345, 2455, 2454835],
+                [32.688980, -117.192040, "", 4525, 423545, 24354, 54, 2552345, 2455, 2475435],
+                [45.786490, -108.526600, "", 4525, 423545, 24354, 54, 2552345, 2455, 2456435],
+                [45.796180, -108.535652, "", 4525, 423545, 24354, 54, 2552345, 2455, 2455435],
+                [45.774320, -108.494370, "", 4525, 423545, 24354, 54, 2552345, 2455, 2445435],
+                [45.777062, -108.549835, "red-dot.png", 4525, 423545, 24354, 54, 25523345, 2455, 245435]
             ])
```

Comment:
Its fine - we have another ticket which is a more general "move testData out of widgets" comment.  And one we can now do cleanly with the unit test framework in place.